### ISSUE TITLE
Missing symbol applicable accessibility - private_protected

### ISSUE THREAD

**Issue by tonyhallett** [MicrosoftDocs/visualstudio-docs#2865]
The private protected access modifier is valid in C# version 7.2 and later.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b402d2e0-a53a-b61a-4da8-b5ee766d9301
* Version Independent ID: 612695df-5934-d967-d709-500ce02b1395
* Content: [.NET Naming Conventions For EditorConfig files - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-naming-conventions?view=vs-2017#feedback)
* Content Source: [docs/ide/editorconfig-naming-conventions.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/ide/editorconfig-naming-conventions.md)
* Product: **visual-studio-windows**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**
